crates/re_query/src/entity_view.rs

```diff
@@ -361,15 +370,17 @@ where
     /// Helper function to produce an `EntityView` from rust-native `field_types`
     #[inline]
     pub fn from_native2<C>(
-        primary: (&[InstanceKey], &[Primary]),
-        component: (&[InstanceKey], &[C]),
+        primary: (&'a [InstanceKey], &'a [Primary]),
+        component: (&'a [InstanceKey], &'a [C]),
     ) -> Self
     where
-        C: Component + 'static,
+        C: Component + re_types::Component + Clone + 'a + 'static,
```

Comment:
why the `+ 'static`?

Reply:
Because it won't compile otherwise :shrug: